sorc/build_ww3prepost.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/sh
 set -x
 
+# Determine which switch to use 
+ww3switch=model/esmf/switch
+while getopts "c" option; do
+  case "${option}" in
+    c)
```

Comment:
When does -c get used? There is also no hook in build_all to turn it on. Does it need to be coordinated with build_ufs somehow?

Reply:
@WalterKolczynski-NOAA thanks for finding this... it was passed in before at one point, I guess it got messed up in a bad merge when the -c was removed from the build_all.  I'll have to re-think this now.

Reply:
Maybe use `-m` for 'meshcap' in `build_all.sh` instead, since `-c` is taken? Can easily turn -m for build_all into -c for build_ww3prepost.

Reply:
Nevermind, the way you changed it to use APP is better.